index.html

```diff
@@ -194,9 +219,10 @@ <h4>
                 </li>
               </ol>
             </li>
-            <li>If a file type is being blocked due to security considerations,
-            return <a>a promise rejected with</a> with a {{"NotAllowedError"}}
-            {{DOMException}}.
+            <li>If |data|'s {{ShareData/files}} member is present and
+            {{Navigator/canShareFiles()}} returns false when called with the
+            files' extensions and content types, return <a>a promise rejected
```

Comment:
Note: This needs to be spec'ed as a `for each |file:File| in files:`, the check each file's content type separately.